.circleci/config.yml

```
@@ -187,7 +187,7 @@ commands:
       - run: ./.circleci/windows-env.ps1
       - run: node --version
       - run: yarn --version
-      - run: yarn install --frozen-lockfile --non-interactive
+      - run: yarn install --frozen-lockfile --non-interactive --cache-folder ~.cache/yarn
```

Why not `~/.cache-yarn` (here and below)?

```suggestion
      - run: yarn install --frozen-lockfile --non-interactive --cache-folder ~/.cache/yarn
```